### PR TITLE
Fix migration from `v-tooltip` to native `title` in menu bar buttons `ActionEntry`

### PR DESCRIPTION
**Pull request by ShGKme**
#### Migration from `v-tooltip` to native `title` was not complete in `<ActionSingle>`

- Regression of: https://github.com/nextcloud/text/pull/5174/
  - Shows incorrect `title` on `NcAction`
  - Floods console with dozen errors per several seconds
- Removed using directive and fixed `title` text

image

##### 🖼️ Screenshots

Before | After
---|---
image | image

#### Duplication of `t` definition

In some components translation method `t` was added as a computed property `t: () => window.t` or a method. But method `t` is already added globally by prototype monkey-patching: `Vue.prototype.t = t`.

It caused error. Removed as not needed.

image

### 🏁 Checklist

- [x] Code is properly formatted (`npm run lint` / `npm run stylelint` / `composer run cs:check`)
- [x] [Sign-off message](https://probot.github.io/apps/dco/) is added to all commits
- [ ] [Tests](https://github.com/nextcloud/text#-testing-the-app) (unit, integration and/or end-to-end) passing and the changes are covered with tests
- [x] Documentation ([README](https://github.com/nextcloud/text/blob/main/README.md) or [documentation](https://github.com/nextcloud/documentation/blob/master/admin_manual/configuration_server/text_configuration.rst#L2)) has been updated or is not required

### CODE REVIEW

File: src/components/Menu/ActionSingle.vue [hunk 1]
```diff
@@ -98,20 +98,11 @@ export default {
...
 
 		const { class: classes, ...attrs } = bindState
 
-		// do not use tooltip if is a item of action list
-		const directives = isItem
-			? []
-			: [{
-				name: 'tooltip',
-				value: tooltip,
-			}]
```
Comment (ShGKme) [inline src/components/Menu/ActionSingle.vue:113]: Currently `title` (that replaces the tooltip) has `isItem ? undefined : label`, not `tooltip` as a value.

https://github.com/nextcloud/text/blob/4a707d05643c182e114b9c1a44d718b9c26bc877/src/components/Menu/BaseActionEntry.js#L68-L73

Should here `title` also be `tooltip` and not `label`?

Comment (ShGKme) [reply src/components/Menu/ActionSingle.vue:113]: Hmm, I think, yes, it should. It shows the shortcut on desktop

Comment (ShGKme) [reply src/components/Menu/ActionSingle.vue:113]: PR Updated

**Comment by juliushaertl** [comment]
No need for comiting compiled assets here, for text we automated this after the merge.

**Review by juliushaertl — APPROVED** [review]
Thanks for that ❤️

**Comment by ShGKme** [comment]
This seems to be a regression of https://github.com/nextcloud/text/pull/5174

So should also be backported

**Comment by ShGKme** [comment]
2 commits just squashed into one as they both actually fix migrating from `v-tooltip` to `title.

No changes in the code.

**Comment by ShGKme** [comment]
Removing `t` from suggestions was a mistake. Suggestion lists are mounted as separated apps and don't reuse `t` from the text app init.

Returned it 🙈

**Pull request merged**